app/controllers/index.js

```diff
@@ -1,23 +1,22 @@
 /**
  * Module dependencies.
  */
-var mongoose = require('mongoose'),
-    async = require('async'),
-    _ = require('underscore');
+var mongoose = require('mongoose'), //eslint-disable-line
```

Comment:
I'm not sure why you have our `eslint` disabled. 
Please let's enable them and fix all lint errors. @saheedt